pkg/types/types.go

```diff
@@ -57,9 +58,11 @@ type PerformLogProvider interface {
 }
 
 type PerformLog struct {
-	Key           UpkeepKey
-	TransmitBlock BlockKey
-	Confirmations int64
+	Key             UpkeepKey
+	TransmitBlock   BlockKey
+	Confirmations   int64
+	TransactionHash common.Hash
```

Comment:
One goal of the plugin is to not introduce any chain specific dependencies. Either make the transaction hash a native type or create a wrapping type for it.